src/tools/rbd/action/Status.cc

```diff
@@ -148,6 +174,15 @@ static int do_show_status(librados::IoCtx& io_ctx, const std::string &image_name
       }
       std::cout << std::endl;
     }
+
+    if (cache_state.present) {
+        std::cout << "image cache state:" << std::endl;
+        std::cout << "\tpresent: " << cache_state.present
+                  << "  clean: " << cache_state.clean
```

Comment:
Nit: convert bool to nice human-readable string

Reply:
Done